src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -1,119 +1,94 @@
 import sortBy from 'lodash/sortBy';
 import property from 'lodash/property';
-import isEqual from 'lodash/isEqual';
-import PropTypes from 'prop-types';
 import React from 'react';
+import {RouteComponentProps} from 'react-router/lib/Router';
 
 import {isUrl, percent} from 'app/utils';
 import {t} from 'app/locale';
-import withApi from 'app/utils/withApi';
+import AsyncComponent from 'app/components/asyncComponent';
 import UserAvatar from 'app/components/avatar/userAvatar';
 import DeviceName from 'app/components/deviceName';
 import ExternalLink from 'app/components/links/externalLink';
 import GlobalSelectionLink from 'app/components/globalSelectionLink';
 import LoadingError from 'app/components/loadingError';
 import LoadingIndicator from 'app/components/loadingIndicator';
 import Pagination from 'app/components/pagination';
-import SentryTypes from 'app/sentryTypes';
 import TimeSince from 'app/components/timeSince';
-import withOrganization from 'app/utils/withOrganization';
-
-class GroupTagValues extends React.Component {
-  static propTypes = {
-    api: PropTypes.object,
-    group: SentryTypes.Group.isRequired,
-    location: PropTypes.shape({
-      query: PropTypes.object,
-    }),
+import {Group, AvatarUser} from 'app/types';
+
+type RouteParams = {
+  groupId: number;
+  orgId: string;
+  tagKey: string;
+};
+
+type TagValue = {
```

Comment:
On first glance it feels like this should probably live in `app/types`. Also it feels like this has a lot of properties, are these all part of an individual tag value?

Reply:
Yeah, they're what become the fields on the CSV when you export on this page https://sentry.io/organizations/sentry/issues/1341088094/tags/user/